spec/plans/report.fmf

```diff
@@ -62,14 +62,28 @@ description:
         - implemented-by: /tmt/steps/report/junit.py
 
 /polarion:
-    summary: Generate a xUnit file and export it into Polarion
+    summary: Generate an xUnit file and export it into Polarion
     story:
         As a tester I want to review tests in Polarion
         and have all results linked to existing test cases there.
     description: |
-        Create a xUnit file ``xunit.xml`` with test results
-        and Polarion properties so the xUnit can then be
-        exported into Polarion.
+        Write test results into a xUnit file and upload to Polarion.
```

Comment:
> although I'm still not sure about it, however it could probably be considered an acronym

I thought that in English the indefinite article `a` should be used when the pronunciation of the next word starts with a consonant. In this case `eksjůnit` starts with a vowel, so `an` should be used. Is that right? The other way somehow does not go through my mouth :)

Reply:
I see, I missed the part about pronunciation, was always considering only what is written

still missing the one 'a xUnit' -> 'an xUnit' as per suggestion above :)

Reply:
Yeah, good, just wanted to confirm the change is ok. Applied.